endpoints/API_Player.py

```diff
@@ -10,17 +10,30 @@
 import urllib.parse
 import logging
 from conf import api_example_conf as conf
+from utils import interactive_mode
+#from conftest import interactivemode_flag
```

Comment:
How are commented out lines of code slipping through to a PR?

Reply:
Fixed